lib/runner/extensions/event.command.js

```diff
@@ -288,6 +289,18 @@ module.exports = {
                         });
                     });
 
+                // To store error event, but only if needed. Because error in callback of host.execute()
+                // don't show execution errors for async scripts
+                (stopOnScriptError || abortOnError) &&
+                    this.host.on(EXECUTION_ERROR_EVENT_BASE + executionId, function (scriptCursor, err) {
```

Comment:
`once` to store only the first async error.
Also, add a test for multiple async errors.